piety-promoted-v3/modernized/01-chapter-01.adoc

```diff
@@ -538,14 +544,19 @@ but that they may not only be the called, but chosen of the Lord.
 Now, in a degree of my heavenly Father`'s love, do I affectionately bid you farewell,
 desiring that grace, mercy and peace may be multiplied in and among you,
 and conclude your true friend,
+
+[.signed-section-signature]
 ELIZABETH SMITH
```

Comment:
Not sure if Jason was directing you otherwise on this, but my thought would be to keep the ALL CAPS _only for the first appearance of the name_ at the top of each section, so I think this would be better changed to `Elizabeth Smith`

Reply:
Yes, he did say that, I just overlooked it! Sorry!